<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dell_window</class>
 <widget class="QMainWindow" name="Dell_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QComboBox" name="comboBox_3"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Consolas&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Consolas&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pushButton_dell">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Consolas&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="pushButton_2_clear">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
